packages/stark-build/config/build-utils.js

```diff
@@ -90,34 +101,39 @@ function getNbbAssetsConfig() {
 }
 
 /**
- * Returns assets set in .angular-cli.json file of the project.
+ * Returns assets set in angular.json file of the project.
```

Comment:
Can you maybe add a link to the reference where we can check all the different fields of the angular.json? that way it would be easier to understand this...

Reply:
done